web/src/components/jobs/JobTags.tsx

```diff
@@ -0,0 +1,290 @@
+// Copyright 2018-2024 contributors to the Marquez project
+// SPDX-License-Identifier: Apache-2.0
+import * as Redux from 'redux'
+import {
+  Autocomplete,
+  AutocompleteChangeDetails,
+  AutocompleteChangeReason,
+  Checkbox,
+  TextField,
+} from '@mui/material'
+import { Box, createTheme } from '@mui/material'
+import { IState } from '../../store/reducers'
+import { Tag } from '../../types/api'
+import { addJobTag, addTags, deleteJobTag } from '../../store/actionCreators'
+import { bindActionCreators } from 'redux'
+import { connect, useSelector } from 'react-redux'
+import { useTheme } from '@emotion/react'
+import Button from '@mui/material/Button'
+import CheckBoxIcon from '@mui/icons-material/CheckBox'
+import CheckBoxOutlineBlankIcon from '@mui/icons-material/CheckBoxOutlineBlank'
+import Chip from '@mui/material/Chip'
+import Dialog from '@mui/material/Dialog'
+import DialogActions from '@mui/material/DialogActions'
+import DialogContent from '@mui/material/DialogContent'
+import DialogTitle from '@mui/material/DialogTitle'
+import LocalOfferIcon from '@mui/icons-material/LocalOffer'
+import MQText from '../core/text/MqText'
+import MQTooltip from '../core/tooltip/MQTooltip'
+import React, { useState } from 'react'
+import Snackbar from '@mui/material/Snackbar'
+
+interface JobTagsProps {
+  namespace: string
+  jobName: string
+  jobTags: string[]
+}
+
+interface DispatchProps {
+  deleteJobTag: typeof deleteJobTag
+  addJobTag: typeof addJobTag
+  addTags: typeof addTags
+}
+
+type IProps = JobTagsProps & DispatchProps
+
+const JobTags: React.FC<IProps> = (props) => {
+  const { namespace, jobName, jobTags, deleteJobTag, addJobTag, addTags } = props
+
+  const [listTag, setListTag] = useState('')
+  const [openTagDesc, setOpenTagDesc] = useState(false)
+  const [tagDescription, setTagDescription] = useState('No Description')
+  const [selectedTags, setSelectedTags] = useState<string[]>(jobTags)
+
+  const handleButtonClick = () => {
+    setOpenTagDesc(true)
+  }
+
+  const [snackbarOpen, setSnackbarOpen] = useState(false)
+  const theme = createTheme(useTheme())
+
+  const handleTagDescClose = () => {
+    setOpenTagDesc(false)
+    setListTag('')
+    setTagDescription('No Description')
+  }
+
+  const handleTagDescChange = (_event: any, value: string) => {
+    const selectedTagData = tagData.find((tag) => tag.name === value)
+    setListTag(value)
+    setTagDescription(selectedTagData ? selectedTagData.description : 'No Description')
```

Comment:
"No description" is kind of a description in and of itself, is this what you want out of this, would you prefer if it was just empty?

Reply:
Yeah that's my data hat talking - can leave blank.